apps/api/src/lesson/lesson.controller.ts

```diff
@@ -165,6 +169,52 @@ export class LessonController {
     return new BaseResponse({ id, message: "Lesson created successfully" });
   }
 
+  @Post("beta-create-lesson/quiz")
+  @Roles(USER_ROLES.teacher, USER_ROLES.admin)
+  @Validate({
+    request: [
+      {
+        type: "body",
+        schema: createQuizLessonSchema,
+        required: true,
+      },
+    ],
+    response: baseResponse(Type.Object({ id: UUIDSchema, message: Type.String() })),
+  })
+  async betaCreateQuizLesson(
+    @Body() createQuizLessonBody: CreateQuizLessonBody,
+    @CurrentUser("userId") userId: UUIDType,
+  ): Promise<BaseResponse<{ id: UUIDType; message: string }>> {
+    const id = await this.adminLessonsService.createQuizLesson(createQuizLessonBody, userId);
+
+    return new BaseResponse({ id, message: "Lesson created successfully" }) as any;
+  }
+
+  @Patch("beta-update-lesson/quiz")
+  @Roles(USER_ROLES.teacher, USER_ROLES.admin)
+  @Validate({
+    request: [
+      {
+        type: "query",
+        name: "id",
+        schema: UUIDSchema,
+      },
+      {
+        type: "body",
+        schema: updateQuizLessonSchema,
```

Comment:
https://github.com/Selleo/lms-core/pull/291/files#r1886947595
```suggestion
        schema: updateQuizLessonSchema, required: true
```